prover/drivers/kore-driver.md

```diff
@@ -10,6 +10,30 @@ module DRIVER-KORE
   imports PROVER-CORE
   imports K-IO
   imports K-REFLECTION
+  imports VISITOR-SYNTAX
+
+  syntax Pattern ::= classify(Head) [function]
+
+  rule [[ classify(S) => symbol(S) ]]
+       <declaration> symbol S(_) : _ </declaration>
+
+  rule [[ classify(N) => notation(N) ]]
+       <declaration> notation N(_) = _ </declaration>
+
+  syntax Pattern ::= resolveHeads(Pattern) [function]
```

Comment:
This function is needed by all who use our kore-syntax. lets move it to that file